xml/ha_acl.xml

```diff
@@ -218,427 +172,563 @@
       </para>
      </step>
      <step>
       <para>
-       Add additional roles as needed.
+       Confirm the dialog box.
       </para>
      </step>
-    </substeps>
-   </step>
-   <step>
-    <para>
-     Assign your roles to one or multiple ACL targets, which are the
-     corresponding system users. Make sure they belong to the
-     <systemitem class="groupname">haclient</systemitem> group.
+    </procedure>
+   </sect2>
+   <sect2 xml:id="sec-ha-acl-rm-user-crmsh">
+     <title>Removing a User with &crmsh;</title>
+     <para> To remove a user from ACL, replace the placeholder
+     <replaceable>USER</replaceable> with the name of the user: </para>
+    <screen>&prompt.root;<command>crm</command> configure delete <replaceable>USERNAME</replaceable></screen>
+    <para>
+     As an alternative, you can use the <command>edit</command> subcommand:
     </para>
-<screen>&prompt.crm.conf;<command>acl_target</command> &exampleuser_plain; monitor</screen>
-   </step>
-   <step>
-    <para>
-     Check your changes:
-    </para>
-<screen>&prompt.crm.conf;<command>show</command></screen>
-   </step>
-   <step>
+    <screen>&prompt.root;<command>crm</command> configure edit <replaceable>USERNAME</replaceable></screen>
+   </sect2>
+  </sect1>
+
+  <sect1 xml:id="sec-ha-acl-rm-role">
+   <title>Removing an Existing Role</title>
+   <para>
+    The following subsections describe how to remove an existing role in &hawk2; and &crmsh;.
+   </para>
+   <sect2 xml:id="sec-ha-acl-rm-role-hawk2">
+    <title>Removing an Existing Role with &hawk2;</title>
+    <para>To remove a role, proceed as follows:</para>
+    <procedure>
+     <step>
+      <para>
+       Log in to &hawk2;: </para>
+      <screen>https://<replaceable>HAWKSERVER</replaceable>:7630/</screen>
+     </step>
+     <step>
+      <para>
+       In the left navigation bar, select <guimenu>Roles</guimenu>.
+      </para>
+     </step>
+     <step>
+      <para>
+       To remove a role, click the garbage bin icon under
+       the <guimenu>Operations</guimenu> column.
+      </para>
+     </step>
+     <step>
+      <para>
+       Confirm the dialog box. If an error message appears, make sure your
+       role is <quote>empty</quote> and does not reference users.
+      </para>
+     </step>
+    </procedure>
+   </sect2>
+   <sect2 xml:id="sec-ha-acl-rm-role-crmsh">
+    <title>Removing an Existing Role with &crmsh;</title>
     <para>
-     Commit your changes:
-    </para>
-<screen>&prompt.crm.conf;<command>commit</command></screen>
-   </step>
-  </procedure>
+    To remove an existing role, replace the placeholder <replaceable>ROLE</replaceable> with the name
+    of the role:
+   </para>
+   <screen>&prompt.root;<command>crm</command> configure delete <replaceable>ROLE</replaceable></screen>
+   <para>
+    Keep in mind, no user should belong to this role. If there is still a reference to a user in the
+    role, the role cannot be deleted.
```

Comment:
This note probably should be prior to these two subsections as the first thing for users to know about removing of a role?

Reply:
Good idea! I've moved the para and created a note. :+1: